include/mbedtls/asn1write.h

```diff
@@ -316,7 +316,7 @@ int mbedtls_asn1_write_bitstring( unsigned char **p, unsigned char *start,
  * \return          A negative error code on failure.
  */
 int mbedtls_asn1_write_named_bitstring( unsigned char **p,
-                                        unsigned char *start,
+                                        const unsigned char *start,
                                         const unsigned char *buf,
                                         size_t bits );
```

Comment:
Same here with `mbedtls_asn1_write_bitstring` and `mbedtls_asn1_write_named_bitstring`.